kubeflow/core/prototypes/google-cloud-filestore-pv.jsonnet

```diff
@@ -0,0 +1,100 @@
+// @apiVersion 0.1
+// @name io.ksonnet.pkg.google-cloud-filestore-pv
+// @description Creates PV and PVC based on Google Cloud Filestore NFS
+// @shortDescription Creates PV and PVC based on Google Cloud Filestore NFS
+// @param name string Name for the component
+// @optionalParam namespace string null Namespace to use for the components. It is automatically inherited from the environment if not set.
```

Comment:
Do we need a namespace parameter? Can't we just assume its the same as that of the environment. If users want to set it they can directly customize the generated component.

Reply:
This is just following the namespace param convention in all of our other components. We can refactor it out from our components if needed.